apps/wallet/src/ui/app/components/alert/index.tsx

```diff
@@ -1,36 +1,47 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-import { Info16, Check32 } from '@mysten/icons';
-import cl from 'classnames';
+import { Info16, CheckStroke16 } from '@mysten/icons';
+import { cva, type VariantProps } from 'class-variance-authority';
 
 import LoadingIndicator from '_components/loading/LoadingIndicator';
 
 import type { ReactNode } from 'react';
 
-import st from './Alert.module.scss';
+const alertStyles = cva(
+    'rounded-2xl text-pBodySmall font-medium flex flex-row flex-nowrap justify-center items-center py-2 px-2.5 gap-2',
+    {
+        variants: {
+            mode: {
+                warning:
+                    'border-solid border bg-issue-light border-issue-dark/20 text-issue-dark',
+                success:
+                    'border-solid border bg-success-light border-success-dark/20 text-success-dark',
+                loading: 'bg-steel text-white border-warning-dark/20',
+            },
+        },
+        defaultVariants: {
+            mode: 'warning',
+        },
+    }
+);
+
+export interface AlertProps extends VariantProps<typeof alertStyles> {
+    children?: ReactNode;
```

Comment:
I think `children` should be mandatory (even though ReactNode is kind of a catch-all... but realistically we'll never render an Alert without children)